doc/sphinx/source/ensemble.txt

```diff
@@ -0,0 +1,22 @@
+Ensemble Objects
```

Comment:
move to `analysis/ensemble.txt` — these are the docs for the ensemble framework on which the analysis classes rely. Look at the MDA docs https://docs.mdanalysis.org/stable/documentation_pages/analysis_modules.html for analysis that list "building blocks" and then the actual analysis modules.

Reply:
done